File: bestia-server-commons-ex/src/test/java/net/bestia/server/AkkaClusterTest.java
package net.bestia.server;

import org.junit.Assert;
import org.junit.Test;


public class AkkaClusterTest {
	
	@Test(expected = NullPointerException.class)
	@SuppressWarnings("all")
	public void getNodeName_null_throws() {
		AkkaCluster.getNodeName(null);
	}
	
	@Test
	public void getNodeName_ok() {
		String test = AkkaCluster.getNodeName("test");
		Assert.assertEquals("/user/test", test);
		
		test = AkkaCluster.getNodeName("test", "bla");
		Assert.assertEquals("/user/test/bla", test);
	}

}

File: bestia-zoneserver/src/test/kotlin/net/bestia/zoneserver/AkkaClusterTest.kt
package net.bestia.zoneserver

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class AkkaClusterTest {

  @Test
  fun getNodeName_ok() {
    var test = AkkaCluster.getNodeName("test")
    assertEquals("/user/test", test)

    test = AkkaCluster.getNodeName("test", "bla")
    assertEquals("/user/test/bla", test)
  }
}
